packages/aws-cdk/test/integ/cli/cdk-helpers.ts

```diff
@@ -240,6 +240,8 @@ export class TestFixture {
     // Bootstrap stacks have buckets that need to be cleaned
     const bucketNames = stacksToDelete.map(stack => outputFromStack('BucketName', stack)).filter(defined);
     await Promise.all(bucketNames.map(b => this.aws.emptyBucket(b)));
+    // The bootstrap bucket has a removal policy of RETAIN by default, so add it to the buckets to be cleaned up.
+    this.bucketsToDelete.push(...bucketNames);
```

Comment:
Should you call `this.rememberToDeleteBucket()` instead? What's the difference?

Reply:
:shrug: `rememberToDeleteBucket` only takes a single bucket name as an argument, so I'd need to do a `forEach`. And `this.bucketsToDelete` is only actually *used* on line 254 (in this same method). This seemed more straightforward.